gradle.properties

```diff
@@ -1,2 +1,4 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m
+org.gradle.caching=true
```

Comment:
It's useless for local builds. Build cache is good for CI, because on CI, you can grab the caches after build to reuse them in the next run.
```suggestion
```